tests/helpers.py

```diff
@@ -2101,3 +2101,18 @@ def modify_osd_replica_count(resource_name, replica_count):
     params = f'{{"spec": {{"replicas": {replica_count}}}}}'
     resource_name = '-'.join(resource_name.split('-')[0:4])
     return ocp_obj.patch(resource_name=resource_name, params=params)
+
+def check_LSO():
+    """
+    Function to check if LSO is present or not
+    {To be used for BM Cluster}
+
+    Returns:
+        bool: True if LSO present, False if LSO not present
+    """
+    command = "get localvolume -n local-storage "
```

Comment:
The function name and doc say that this is for checking for the existance of the Local Storage Operator, however we are calling get on the `localvolume` here instead. Shouldn't we be checking the status of the LSO? 

We do something similar to this in the [LSO Deployment code](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/deployment/deployment.py#L742-L749)

Reply:
Basically, I am adding a small function to check both for local volumes and local storage operator. This is required to test ```add_capacity``` over Baremetal LSO. We have just started with the work and this comes as one of the basic checks to complete that function.

Reply:
Also, we are not creating a deployment and checking over it. we just want our function ```check_LSO()``` to just verify if it is already up or not for the use-case.

Reply:
What I mean is think that the name `check_LSO` is misleading because you are actually checking the localvolume here and not checking the status of the local storage operator. To my knowledge the localvolume and LSO are not the same thing so we should either change the name and description of this function or also be checking the status of the LSO here, right?

The code from the deployment was to give an example of how we are checking the status of the LSO already.

Reply:
Okay, I fixed the name. Added some more changes regarding supporting ```add_capacity``` over BM LSO. Still not the final thing. Work in Progress. Need Suggestions!

Reply:
If this is work in progress I'm going to add `WIP` to the title so others know